src/app/project/list/list-item/list-item.component.ts

```diff
@@ -54,7 +55,6 @@ export class ListItemComponent implements OnInit {
                 }
             );
         }
-
     }
 
     showChildren(id: string): boolean {
```

Comment:
I will be picky: would it be possible to add in this file as well some method description? even though it is not from you

Reply:
added in be62277